runtime/integration-tests/src/integrations_mock/test_common.rs

```diff
@@ -696,6 +700,164 @@ fn concrete_fungible_ledger_transfers_work() {
         });
 }
 
+fn pause_transaction_storage_event_works(pause: bool) {
```

Comment:
pls make this a `mod tx_pause_tests` as parastaking above. That makes if fold away more easily

Reply:
Sure thing

we probably should separate them into different files as more lines are added. That would be better done in different PR